presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -101,19 +108,22 @@ Date and Time Functions
 
     Returns a timestamp with time zone from ``timestamp`` and ``zone``.
 
-.. function:: from_unixtime(unixtime) -> timestamp
+.. function:: from_unixtime(unixtime) -> timestamp(3)
 
-    Returns the UNIX timestamp ``unixtime`` as a timestamp. ``unixtime`` is the number of seconds since ``1970-01-01 00:00:00``.
+    Returns the UNIX timestamp ``unixtime`` as a ``timestamp(3)``. ``unixtime``
+    is the number of seconds since ``1970-01-01 00:00:00 UTC``.
 
 .. function:: from_unixtime(unixtime, zone) -> timestamp with time zone
 
-    Returns the UNIX timestamp ``unixtime`` as a timestamp with time zone
-    using ``zone`` for the time zone. ``unixtime`` is the number of seconds since ``1970-01-01 00:00:00``.
+    Returns the UNIX timestamp ``unixtime``, a double as a timestamp with
```

Comment:
This seems unrelated to timestamp with precision. Please remove.

Reply:
This add UTC as requested by @martint

Reply:
Please only add "UTC" and don't make any formatting changes. Adding data types is unrelated to timestamp precision and should be done / discussed separately.